docs/Developer-Docs/Managing-Conda-Lock-File.rst

```diff
@@ -0,0 +1,21 @@
+Managing the Conda Lock File
+------------------------------
+
+The default conda environment set by ``build-setup.sh`` uses the `lock file ("*.conda-lock.yml") <https://github.com/conda-incubator/conda-lock>`_ at the top of the repository.
+This file is derived from the normal conda requirements file (``*.yaml``) also located at the top-level of the repository.
+
+Updating Conda Requirements
+===========================
+
+If developers want to update the requirements file, they should also update the lock file accordingly.
+This is done by two methods:
```

Comment:
```suggestion:
There are two different methods:
```

Reply:
nit: for some reason i thought you had to run both for a second, this seems a little more clear.